docs/manual.rst

```diff
@@ -271,10 +270,6 @@ collection to get back to the beginning.
 Collection indexing
 -------------------
 
-.. admonition::
-
-   New in version 1.1.6
-
 Features of a collection may also be accessed by index.
 
 .. code-block:: pycon
```

Comment:
Another one here.

Reply:
This one looks okay to me. The following code block does have python prompts and "pycon" is intended.

We have inconsistent use of code-block and sourcecode directives, but that's out of scope.

Reply:
Sounds good, thanks for getting this MR ready :+1: